content/markdown/04-intro-to-stats/04-08-linear-regression.Rmd

```diff
@@ -0,0 +1,468 @@
+2. <strong>Residuals</strong> -- These are the residuals. Recall that a <em>residual</em> is the distance from a single point to the regression line at the same x-location. Here's a visual of that: 
+
+```{r}
+ggplot(data = df, mapping = aes(x = x, y = y)) + 
+  geom_smooth(method = "lm", formula = y ~ x, colour = "red", se = FALSE) +
+  ggthemes::theme_base() +
+  geom_point(data = df %>% dplyr::filter(
+    x %in% c(22, 45, 89)
+  ), size = 3, colour = "black", fill = "purple", shape = 21) +
+  geom_line(data = data.frame(x = c(22, 22), y = c(
+    predict(mod, data.frame(x = 22)), df$y[which(df$x == 22)]
+  )), aes(x = x, y = y)) + 
+  geom_line(data = data.frame(x = c(45, 45), y = c(
+    predict(mod, data.frame(x = 45)), df$y[which(df$x == 45)]
+  )), aes(x = x, y = y)) + 
+  geom_line(data = data.frame(x = c(89, 89), y = c(
+    predict(mod, data.frame(x = 89)), df$y[which(df$x == 89)]
+  )), aes(x = x, y = y)) 
+```
+
+Each of the lines here going from the points to the the regression line itself are "residuals", or "residual error", the amount of "error" the linear model produces in it's prediction for a given x-value.
+
+Ok back to our <code>summary()</code> call: 
+
+```{r}
+summary(mod)
+```
+
+the residuals here, by definition, have a mean of zero, and a "perfecty" minimized set of residuals would mean that the median value is zero, which will not really happen, but we want the median to be as close to zero as possible. We also want our minimum and maximum values to be approximately equal in magnitude. 
+
+3. <strong> Coefficients </strong> -- Possibly the part that we'll look at the most, these values are here shown as representing $\beta_0$ and $\beta_1$. So the intercept is the actual y-intecept (remember that $\beta_0$ is the y-intercept), and the $\beta_1$ is the estimate of the coefficient on $x$. We assess whether or not a particular predictor variable is significantly associated with the outcome variable via the "Pr(>|t|)" statement of the p-value. For our purposes, this will be useful, but I encourage you to interpret these p-values with caution, especially when there are multiple variables. 
+
+For our consideration, we take the <em>Estimate</em> as the actual estimated value for the $\beta$s and then the standard errors (SE), which define accuracy of beta coefficients. A larger SE indicates a less certain beta estimate. We then see the t-statistic and the associated p-value, which speak to the statistical significance of the beta coefficients.
+
+We have to recall that for our regression, we are using the $H_0$ that the estimate is zero, which denotes no effect from said coefficient. If any coefficient has a p-value < 0.05, we can infer that the estimate is significantly different from zero. However, this says nothing about the <em>strength</em> of that interaction. 
+ 
+If we cared to construct a confidence interval around our beta values, we could do so with $\beta +/- 1.96 * SE_{\beta}$, which would give us our 95% CI. 
```

Comment:
only if the sample size is large- otherwise the t-distribution should be used to calculate CIs not the standard normal